### ISSUE TITLE
Invalid flow specified

### ISSUE THREAD

**Issue by martijnaikema** [rstrouse/ESPSomfy-RTS-HA#23]
I successfully installed hardware and software and am able to control my shades via the up address. 

After installing the home assistant ESPSomfy via Hacs I am trying to configure it. When entering ip (auto found) and authentication I get the error:

Invalid flow specified

any directions in how to solve this issue?

**Comment by rstrouse** [comment]
Restart HA and reload your browser.  This is acting like it did not find the integration.  Perhaps reload the integration as well by clicking the 3 dots and choose reload.

**Comment by martijnaikema** [comment]
Tried it all. Unfortunately no succes yet. Also tried installing different versions but no luck.

**Comment by rstrouse** [comment]
That is very odd.  What version of HA are your running?

**Comment by rstrouse** [comment]
In hacs try installing the main repository.  Perhaps this has something to do with translations.

**Comment by martijnaikema** [comment]
> In hacs try installing the main repository. Perhaps this has something to do with translations.

Am running Home Assistant 2023.9.2 in container. 

I just changed my language settings to English and now it seems to work. Thanx for your help.

**Comment by martijnaikema** [comment]
I am running Home Assistant 2023.9.2 in container. 

Just updated my language settings to English (was Dutch) and then I could configure. Thanx for your help!

**Comment by mkruiver** [comment]
I am currently experiencing the same "invalid flow specified" with HA Supervised. Running 2023.10.1 in English.

**Comment by rstrouse** [comment]
For some reason, HA does not restart anymore when a new HACS component is added.  It does when you update a component and it does in hassOS.  The custom components are also not completely loaded until a restart.

**Comment by irrwisch1** [comment]
I'm currently having the same issue. Trying to add the discovered ESP Somfy device either gives me an "invalid flow specified" or an "Unknown error occurred".
I tried rebooting my hassIO vm several times and switched language to english, but that did not help.
Any ideas?
My HA is updated to the latest version (2023.11.1)

Thanks

**Comment by hank99scorpio** [comment]
Same problem here:
 Core 2023.11.3
Supervisor 2023.11.6
Operating System 11.1
Frontend 20231030.2

rebooted, reinstalled ESPSomfy RTS-HA

**Comment by miqmago** [comment]
Same here, uninstalled addon, changed language of user and of house to english, redownloaded the addon, restarted and changed back language to catalan.

The only way was to remove security user and password, reboot espsomfy device, pair with IP (not with hostname).
Then I added again user and password, went to integration settings and in hubs section (not in discovered section) I changed to hostname and added user and password.

It seemed to work, but when reloading then it appears again as not configured.

The only way to make it work at this moment is without user / password

**Comment by Tornix242** [comment]
Hi there,

First thank you so much for this incredible work!!!

I just setted up everything. Shades a correctly moving from the webpage.
On my side HA integration do not work at all, even without user/password and both HA and user language settings in English.

I have either "Invalid flow specified" or "Unknown error occurred".

The path to setup was challenging (and of course I messed up a shade before getting working😄).
I hope you ladies and gentlemen will be able to help us, and I'm willing to contribute if I can!
